### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by dispersik**

### CODE REVIEW

File: homework4-1.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+func (r Rectangle) area() float64 {
+	return r.a*r.b
+}
+
+
+func main() {
+
+	var s Figure = Square{1}
+	var c Figure = Circle{1}
+	var r Figure = Rectangle{3,4}
```
Comment (wshaman) [inline homework4-1.go:55]: This case is pain. Generally, it's bad practice to create struct objects with unnamed params.
It shouldn't be fixed due to task description, just keep in mind

**Review by wshaman — APPROVED** [review]
:+1:

File: homework4-2.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+type People []Person
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p People) Less(i, j int) bool {
+	//Firstly, compare firstName n lastName strings
```
Comment (VolhaVetrava) [inline homework4-2.go]: Comment should be written above function declaration

**Review by VolhaVetrava — APPROVED** [review]

File: homework4-2.go (hunk 2 above)
Comment (dispersik) [reply homework4-2.go]: I have fixed that issue

**Pull request merged**